### PR TITLE
update release stream jobs for device-snmp-go

### PR DESCRIPTION
**Pull request by dweomer**
This corrects the job definition for `device-snmp-go` to have jobs for both `master` and `edinburgh` branches.

Signed-off-by: Jacob Blain Christen <jacob.blain.christen@intel.com>

### CODE REVIEW

**Review by jamesrgregg — DISMISSED** [review]
LGTM

**Review by lranjbar — COMMENTED** [review]
LGTM template wise. Jacob and I would like to talk about this in the Thursday DevOps meeting.

Please hold on merge for now.

**Comment by dweomer** [comment]
Please wait for these to be merged before merging this:
- https://github.com/edgexfoundry/device-snmp-go/pull/14
- https://github.com/edgexfoundry/device-snmp-go/pull/15
- https://github.com/edgexfoundry/device-snmp-go/pull/16

**Review by jamesrgregg — APPROVED** [review]
LGTM - all changes merged on device-snmp-go repo

**Pull request merged**
